qemu/tests/win_heavyload.py

```diff
@@ -7,14 +7,14 @@
 except ImportError:
     from virttest import aexpect
 
-from autotest.client.shared import error
+from virttest import error_context
```

Comment:
Should move this line to where after L11?

Reply:
good,  updated, thanks!